stablehlo/tests/interpret_clamp.mlir

```diff
@@ -0,0 +1,74 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: clamp_op_test_si64
+func.func @clamp_op_test_si64() -> tensor<3xi64> {
+  %0 = stablehlo.constant dense<[1, 5, -5]> : tensor<3xi64>
```

Comment:
Can you change the SSA variables to `%min`, `%operand`, `%max`, `%result`?

Reply:
Very good advice!